2/contrib/s2i/run

```diff
@@ -186,12 +186,23 @@ if [ ! -e ${JENKINS_HOME}/configured ]; then
     touch ${JENKINS_HOME}/configured
 else  
   if [ ! -z "${OVERRIDE_PV_CONFIG_WITH_IMAGE_CONFIG}" ]; then
-    echo "Overriding jenkins config.xml stored in ${JENKINS_HOME}/config.xml"
-    rm -f ${JENKINS_HOME}/config.xml
-	
-    create_jenkins_config_xml
+    echo "Overriding jenkins configuration stored in ${JENKINS_HOME}"
+
+    # remove any files/directories in $JENKINS_HOME that are specified in $image_config_dir, 
+    # otherwise the two will get merged when cp is run below
+    for f in ${image_config_dir}/*; do
+      fileName=${f#${image_config_dir}'/'}
+      if [ -e "${JENKINS_HOME}/${fileName}" ]; then
+        echo "---> Removing ${JENKINS_HOME}/${fileName}..."
+        rm -rf "${JENKINS_HOME}/${fileName}"
+      fi
+    done
+
+    # re-running install_plugins when OVERRIDE_PV_CONFIG_WITH_IMAGE_CONFIG is set is not currently implemented
```

Comment:
this comment is very strange.  this would never be implemented, it's why `OVERRIDE_PV_PLUGINS_WITH_IMAGE_PLUGINS` exists.